src/ws/login.js

```diff
@@ -239,9 +242,6 @@
             brand("brand", "Cockpit");
         }
```

Comment:
So before you still saw the badge/branding, but no login form. With this change, what hides the login form for non-met requisites? I. e. I'm sure you tested this, but I don't understand why this code moved. Maybe explain in the commit message?

Reply:
The login screen gets extremely jumbled with that information there. If you're using an old/broken/unsupported browser, not much other information matters until the browser situation is fixed.